### ISSUE TITLE
servcheck plugin sends emails to email lists but as separate emails not one email to all recipients

### ISSUE THREAD

**Issue by mohabuhtig** [Cacti/plugin_servcheck#5]
Dear Team,
First of all, I would like to express my gratitude and thank you very very much for providing this plugin "servcheck" as a replacement to 'webseer". the "servcheck" is more accurate and easy to utilize and accordingly monitor services.
Much appreciated once again.

I have been facing the following issue and I appreciate your support.

**Describe the bug**
Introduction:
servcheck plugin sends emails to email lists.
if the email list contains multiple email addresses separated by comma, the following bug occurs.

servcheck plugin sends emails to email list (which contains multiple email addresses separated by comma) based on the following cases/events:

**Undesired behavior:**
Individual emails are generated and sent to each email address separately, the email would look like it has been sent to one email only.
This is **undesirable** behavior because it does NOT:
a) Provide the information to the recipients about who receives the email as well.
b) Allow automatic assignment of the ITSM incidents when the email is sent to the email list that contains ITSM email address and the support engineer email address, as the email is received by the ITSM without the email address of the support engineer, it is not possible to automatically assign the incident to concerned engineer.

**Desired behavior:**
One email is generated and sent to all email addresses included in the email list, the email would look like it has been sent to one email only.
This is **desirable** behavior because it will:
a) Provide the information to the recipients about who receives the email as well.
b) Provide more control when the email is sent to the email list that contains ITSM email address and the support engineer based on which the ITSM automatically categorizes and assigns the incident to required support engineer.
c) Unified behavior when sending the email notification

**Plugin:**
 - Version: 0.1
 - Source: github cacti

![servcheck-email-notification](https://github.com/Cacti/plugin_servcheck/assets/124117691/4d9875c8-1504-4088-9ccc-601127ffeb67)

**Comment by mohabuhtig** [comment]
Much appreciated Petr.
I appreciate your support and time.

**Comment by mohabuhtig** [comment]
No, the problem is that separate emails are being sent to each email address with the same notification list.

e.g. email notification list = "Network Team" contains 2 email address with comma separated e.g.:
network.engineer@abc.com, servicedesk@abc.com

the email is required to be sent as one single email to both emails at the same time so that in the "To:" email field, 2 email address are required to be listed, if this is achieved, then I can configure the ServiceDesk system to find network.engineer email and assign the ticket to him.

**Comment by mohabuhtig** [comment]
at present, the email is sent to servicedesk@abc.com as one email 
and another email is sent to network.engineer@abc.com

This does not allow me to automatically assign the ticket to the network engineer as the email received in the ServcieDesk does not include the network.engineer email address

I hope this is clear

**Comment by mohabuhtig** [comment]
Below are the 2 individual emails received:
1. to the service desk
2. to the support team.
![Screenshot 2024-06-05 174444](https://github.com/Cacti/plugin_servcheck/assets/124117691/b88a4c4b-396e-41a7-8170-e0f44a3eea89)
![Screenshot 2024-06-05 174636](https://github.com/Cacti/plugin_servcheck/assets/124117691/557bb7b4-6c3c-4061-8337-f501be722d9b)

**Comment by mohabuhtig** [comment]
I want to see the emails sent as one email containing both email addresses e.g.

![Screenshot 2024-06-05 174444-a](https://github.com/Cacti/plugin_servcheck/assets/124117691/ec470205-0de6-41f1-a1e8-7dd12a689d38)

**Comment by xmacan** [comment]
It is small change, please change few lines:
https://github.com/Cacti/plugin_servcheck/pull/6/files

**Comment by mohabuhtig** [comment]
Much appreciated, I have done the changes as advised and will test the system and update you.
Thanks Petr.
Much appreciated.
Regards,
MSS

**Comment by mohabuhtig** [comment]
Tested, it is working as expected :)
👍 
Thanks once again.
